proto/cosmos/authz/v1beta1/query.proto

```diff
@@ -50,8 +57,24 @@ message QueryGranterGrantsRequest {
 
 // QueryGranterGrantsResponse is the response type for the Query/GranterGrants RPC method.
 message QueryGranterGrantsResponse {
-  // authorizations is a list of grants granted for grantee by granter.
-  repeated cosmos.authz.v1beta1.Grant grants = 1;
+  // grants is a list of grants granted by the granter.
+  repeated GrantAuthorization grants = 1;
```

Comment:
This is still proto-breaking, we should revert back to `repeated Grant` here

Reply:
Yeah I changed it from `Grant` to `GrantAuthorization` because the latter contains both the grantee and granter addresses. Without that information the user won't know which addresses the use gave which grants to or equally from which addresses it's received what grants from. I think this is pretty important information.

Reply:
That makes sense. Could we instead add a new field `full_grants` or `detailed_grants`?

I don't think it's worth bumping to v1beta2 for this change, so let's make it additive. We can also add the `deprecated` proto tag to the old field.

Reply:
Ok I've done that

Reply:
Hmm actually was the initial PR ever released> https://github.com/cosmos/cosmos-sdk/pull/10326. It looks like it's still unreleased according to the changelog

Reply:
Oooh nice find! Yeah you're right it was never released: https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/proto/cosmos/authz/v1beta1/query.proto

Cool, let's do `GrantAuthorization` everywhere then, it's way cleaner